reloj-checador/app/views/reports/average_time.html.erb

```diff
@@ -0,0 +1,87 @@
+<h1 class="average-title-attendances">Average time check-in/check-out </h1>
+  <div class="container">
+    <h5 class="average-title-check">Check in</h5>
+    <div class="row">
+      <div class="col-sm">
+        <p class="average-layout">Week of the day</p> 
+      </div>  
+      <div class="col-sm">
+        <div class="average-dropdown">
+          <button class="average-dropbtn">Dropdown</button>
+            <div class="average-dropdown-content">
+              <a href="#">Link 1</a>
+              <a href="#">Link 2</a>
+              <a href="#">Link 3</a>
+            </div>
+        </div>
+      </div>
+      <div class="col-sm">
+        <div class="average-space-button">
+          <button class="average-save">Open</button>
+        </div>
+      </div>
+    </div>
+    <div class="row pb-4">
+      <div class="col-sm">
+        <p class="average-layout">Company</p> 
+      </div>    
+      <div class="col-sm">
+        <div class="average-dropdown">
+          <button class="average-dropbtn">Dropdown</button>
+            <div class="average-dropdown-content">
+              <a href="#">Link 1</a>
+              <a href="#">Link 2</a>
+              <a href="#">Link 3</a>
+            </div>
+        </div>
+      </div>   
+      <div class="col-sm">
+        <div class="average-space-button">
+          <button class="average-save">Donwload</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <h5 class="average-title-check">Check out</h5>
+    <div class="row">
+      <div class="col-sm">
+        <p class="average-layout">Week of the day</p> 
+      </div>  
+      <div class="col-sm">
+        <div class="average-dropdown">
+          <button class="average-dropbtn">Dropdown</button>
+            <div class="average-dropdown-content">
```

Comment:
If this is a dropdown I suggest using Rails `select_tag` https://guides.rubyonrails.org/v2.3/form_helpers.html#the-select-and-option-tags

If this is just a placeholder and you plan on removing it later, then ignore.